arrow-libs/optics/arrow-optics-compose/src/commonMain/kotlin/arrow/optics/Copy.kt

```diff
@@ -0,0 +1,29 @@
+package arrow.optics
+
+import androidx.compose.runtime.MutableState
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.update
+
+/**
+ * Modifies the value in this [MutableState]
+ * by applying the function [block] to the current value.
+ */
+public inline fun <T> MutableState<T>.update(block: (T) -> T) {
```

Comment:
I think we should note this method is not thread safe. It is only called on MainThread

Reply:
Thanks, I was not aware of this problem!